tasks/sep/006-2-foundation-guardian-updates/SignFromJson.s.sol

```diff
@@ -0,0 +1,537 @@
+        // require(councilOwners.length == foundationOwners.length, "checkSecurityCouncilSafe-200");
+        if (councilOwners.length != foundationOwners.length) {
+            // this should be silenced once the previous runbook is completed
+            console.log(
+                "\x1b[31m TEMP ERROR: checkSecurityCouncilSafe-100",
+                councilOwners.length,
+                foundationOwners.length,
+                "\x1b[0m"
+            );
+        }
+        for (uint256 i = 0; i < councilOwners.length; i++) {
+            // require(foundationSafe.isOwner(councilOwners[i]), "checkSecurityCouncilSafe-201");
+            if (!foundationSafe.isOwner(councilOwners[i])) {
+                // this should be silenced once the previous runbook is completed
+                console.log("\x1b[31m TEMP ERROR: checkSecurityCouncilSafe-101", councilOwners[i], "\x1b[0m");
+            }
+        }
+
+        // See sepolia.json for config values being verified:
+        // https://github.com/ethereum-optimism/optimism/pull/10224/files
+        // The SecurityCouncilSafe should have a treshold of 2
+        require(securityCouncilSafe.getThreshold() == 2, "checkSecurityCouncilSafe-301");
+    }
+
+    function checkUpgradeOwnerSafe() internal view {
+        console.log("Running assertions on the UpgradeOwnerSafe");
+        require(upgradeOwnerSafe.getThreshold() == 2, "checkUpgradeOwnerSafe-100");
+        require(upgradeOwnerSafe.getOwners().length == 2, "checkUpgradeOwnerSafe-200");
+        require(upgradeOwnerSafe.isOwner(address(foundationSafe)), "checkUpgradeOwnerSafe-300");
+        require(upgradeOwnerSafe.isOwner(address(securityCouncilSafe)), "checkUpgradeOwnerSafe-400");
+    }
+
+    /// @notice Checks the correctness of the deployment
+    function _postCheck(Vm.AccountAccess[] memory accesses, SimulationPayload memory /* simPayload */ )
+        internal
+        view
+        override
+    {
+        console.log("Running post-deploy assertions");
+
+        checkStateDiff(accesses);
+        checkSemvers();
+
+        checkSystemConfig();
+        checkL1CrossDomainMessenger();
+        checkL1StandardBridge();
+        checkL2OutputOracle();
+        checkOptimismMintableERC20Factory();
+        checkL1ERC721Bridge();
+        checkOptimismPortal();
+        checkProtocolVersions();
+        checkSuperchainConfig();
+        checkSecurityCouncilSafe();
+        checkLivenessModule();
+        checkLivenessGuard();
+        checkDeputyGuardianModule();
+        checkUpgradeOwnerSafe();
+
+        console.log("All assertions passed!");
+    }
+
+    function getCodeExceptions() internal view override returns (address[] memory) {
+        // Safe owners will appear in storage in the LivenessGuard when added
+        address[] memory securityCouncilSafeOwners = securityCouncilSafe.getOwners();
+        address[] memory shouldHaveCodeExceptions = new address[](7 + securityCouncilSafeOwners.length);
```

Comment:
Same problem I had in #176, I added your suggested check so this should fail after that is merged.

Reply:
This is now resolved